docs/openapi.yml

```diff
@@ -1028,6 +1028,10 @@ components:
     newRun:
       type: object
       properties:
+        runId:
```

Comment:
_Minor:_ Thoughts on naming the ID for a run `externalId` to make it explicit that it's user provided?

Reply:
I would tend to call it runId here since it becomes the actual ID for the run. I would keep externalId to add in the metadata the actual external id defined by a scheduler. That one could be numerical or a string and may not be a uuid.
In this case, it is provided in the request by the client, so I think it is clear it is user provided. Just like Job and Dataset ids are user provided.

Reply:
Yep, that totally makes sense. Since it will be the ID of the run returned to the caller, should we keep naming in sync? That is, in the request it's `runId`, but for the response it's just `id`. For example,

**Request**
```
POST /api/v1/namespaces/my-namespace/jobs/my-job/runs
{
  "runId": "cbd759f7-1948-4013-a14b-7043c0d06f35", <-- use 'id' here to match response?
  "args": {...}
}
```

**Response**
```
{
  "id": "cbd759f7-1948-4013-a14b-7043c0d06f35",
  "args": {...}
}
```

Reply:
that makes sense, I'll change to id for consistency.

Reply:
Thanks! 👍